.github/workflows/omero_plugin.yml

```diff
@@ -30,6 +31,30 @@ jobs:
           repository: ome/omero-test-infra
           path: .omero
           ref: ${{ secrets.OMERO_TEST_INFRA_REF }}
+      - uses: actions/setup-python@v2
```

Comment:
Would this do what you need as a separate workflow?

Reply:
I have not tried it to be honest, time was not on my side :-)

Reply:
Ok. Pushed.

Reply:
Interestingly this is now failing with:

```
copy-client:
Created dir: /home/runner/work/openmicroscopy/openmicroscopy/dist/lib/client
:: problems summary ::
:::: WARNINGS
		[NOT FOUND  ] io.netty#netty-transport-native-epoll;4.1.46.Final!netty-transport-native-epoll.jar (0ms)
	==== user-maven: tried
	  /home/runner/.m2/repository/io/netty/netty-transport-native-epoll/4.1.46.Final/netty-transport-native-epoll-4.1.46.Final.jar
		[NOT FOUND  ] org.kohsuke#file-leak-detector;1.13!file-leak-detector.jar (0ms)
	==== user-maven: tried
	  /home/runner/.m2/repository/org/kohsuke/file-leak-detector/1.13/file-leak-detector-1.13.jar
		::::::::::::::::::::::::::::::::::::::::::::::
		::              FAILED DOWNLOADS            ::
		:: ^ see resolution messages for details  ^ ::
		::::::::::::::::::::::::::::::::::::::::::::::
		:: io.netty#netty-transport-native-epoll;4.1.46.Final!netty-transport-native-epoll.jar
		:: org.kohsuke#file-leak-detector;1.13!file-leak-detector.jar
		::::::::::::::::::::::::::::::::::::::::::::::
```

Reply:
Interesting, `file-leak-detector` is a dependency from the `test-suite` component of ome/bioformats for detecting unclosed file handles.

I suspect it is being pulled as a result of https://github.com/ome/ZarrReader/blob/20ccbe52ff68bb8f7af0ed460c6cc29b0e31ffb8/pom.xml#L124-L127. Assuming nothing depends on it (which arguably would be a bug), either this dependency should be removed completely from the POM or it should be declared within the `test` scope`.

Reply:
Adding ``<scope>test</scope>`` leads to an error when running ``mvn install``
```
ZarrReader.java:[640,15] cannot find symbol
  symbol:   variable StringUtils
```